js/html/shared/HTML.svelte

```diff
@@ -1,27 +1,34 @@
 <script lang="ts">
 	import { createEventDispatcher } from "svelte";
+
 	export let elem_classes: string[] = [];
 	export let value: string;
 	export let visible = true;
-	export let min_height = false;
+	export let min_height: number | undefined = undefined;
+	export let loading_status: { status: string } | undefined = undefined;
 
 	const dispatch = createEventDispatcher<{ change: undefined }>();
 
 	$: value, dispatch("change");
+
+	const css_units = (dimension_value: string | number): string => {
+		return typeof dimension_value === "number"
+			? dimension_value + "px"
+			: dimension_value;
+	};
```

Comment:
Perhaps good to refactor this out and put in `@gradio/utils`, so we don't need to duplicate in Markdown.svelte and here: https://github.com/gradio-app/gradio/blob/1fef9d9a26f0ebce4de18c486702661f6539b1c6/js/atoms/src/Block.svelte#L23